core/src/runtime.ts

```diff
@@ -22,17 +23,20 @@ export interface RegisteredPlugin {
 
 export const createCapacitor = (win: WindowCapacitor): CapacitorInstance => {
   const cap: CapacitorInstance = win.Capacitor || ({} as any);
+  const capPlatforms: CapacitorPlatformsInstance = win.CapacitorPlatforms;
 
   const Plugins = (cap.Plugins = cap.Plugins || ({} as any));
 
   const webviewServerUrl =
     typeof win.WEBVIEW_SERVER_URL === 'string' ? win.WEBVIEW_SERVER_URL : '';
 
-  const getPlatform = () => getPlatformId(win);
+  const defaultGetPlatform = () => getPlatformId(win);
```

Comment:
Not sure I follow why we need to have a `currentPlatform` and rename some of the apis to `default`